include/struct_defs/struct_0205964C.h

```diff
@@ -9,22 +9,22 @@
 
 #include <nitro/math.h>
 
-typedef struct {
-    UnkStruct_02072014 * unk_00[4];
+typedef struct FieldCommunicationManager {
+    TrainerCard * trainerCard[4];
     u8 unk_10[4];
     u8 padding_14[4];
-    UnkStruct_0203CDB0 * unk_18;
+    FieldSystem * fieldSys;
```

Comment:
```suggestion
    FieldSystem * fieldSystem;
```

Reply:
I think `fieldSys` is fine here; we already use `battleSys` in battle code.